src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx

```diff
@@ -222,7 +236,7 @@ export const CheckoutPayment = ({
   const [isDataLoading, setIsDataLoading] = useState(false)
   const [isError, setIsError] = useState(false)
   const [is3dsError, setIs3dsError] = useState(false)
-  const [isPaymentConnected, setIsPaymentConnected] = useState(false)
+  const isPaymentConnected = localStorage.getItem('paymentConnected')
```

Comment:
How is this intended to work? The way we know if payment is connected is by reading the status from quote cart, no?